src/common/include/gudhi/math.h

```diff
@@ -0,0 +1,23 @@
+/*    This file is part of the Gudhi Library - https://gudhi.inria.fr/ - which is released under MIT.
+ *    See file LICENSE or go to https://gudhi.inria.fr/licensing/ for full license details.
+ *    Author(s):       Vincent Rouvreau
+ *
+ *    Copyright (C) 2020 Inria
+ *
+ *    Modification(s):
+ *      - YYYY/MM Author: Description of the modification
+ */
+
+#ifndef MATH_H_
+#define MATH_H_
+
+#include <boost/math/constants/constants.hpp>
+
+namespace Gudhi {
+
+// In wait of C++20 std::numbers::pi with #include <numbers>
+static constexpr double PI = boost::math::constants::pi<double>();
```

Comment:
I think in another branch we said we might as well use the boost constants directly, without a gudhi wrapper.

Reply:
Done on https://github.com/GUDHI/gudhi-devel/pull/397/commits/3a06eff2f6345133270ddc50b625139761c06df6